student.rb

```diff
@@ -1,9 +1,12 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name = 'unknown', parent_permission: true)
-    super(age, name, parent_permission)
+    super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    classroom.add_student(self)
```

Comment:
- [ ] Please define a method classroom that serves as the 'belongs-to' side (a student belongs to a classroom). Look at this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) with examples. The classroom method will be identical to the owner method in the example.